blockly-svelte/README.md

```diff
@@ -0,0 +1,24 @@
+[Home](../README.md)
+
+# blockly-svelte-sample [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This sample shows how to load Blockly in a [Svelte](https://svelte.dev) project.
+
+## Running the sample
+
+### Installation
+
+```
+npm install
+```
+
+### Running
+
+start [Rollup](https://rollupjs.org) by:
+```bash
+npm run dev
+```
+
+### Browse
+
+Navigate to  [http://localhost:5000/](http://localhost:5000/)
```

Comment:
Quick question, do you know how to configure the dev server to use a different port here? I ask because I've tried to setup all dev servers to use port 3000 making it easier to test each sample without having to change the address.

Reply:
used update the sample to use port 3000